docs/natspec-format.rst

```diff
@@ -36,17 +36,16 @@ tools.
 Documentation Example
 =====================
 
-Documentation is inserted above each ``class``, ``interface`` and
-``function`` using the Doxygen notation format.
-
-Note: a ``public`` state variable is equivalent to a ``function``
+Documentation is inserted above each ``contract``, ``interface``,
+``function``, and ``event`` using the Doxygen notation format.
+A ``public`` state variable is equivalent to a ``function``
 for the purposes of NatSpec.
 
 -  For Solidity you may choose ``///`` for single or multi-line
    comments, or ``/**`` and ending with ``*/``.
 
 -  For Vyper, use ``"""`` indented to the inner contents with bare
-   comments. See `Vyper
+   comments. See the `Vyper
```

Comment:
Might this newline cause problems with the link rendering?

Reply:
Is it not rendering correctly? It was when I checked

Reply:
I didn't check, i just saw the line break and wondered if it might be a problem